doc/modules/multiclass.rst

```diff
@@ -64,20 +64,65 @@ Below is a summary of the classifiers supported by scikit-learn
-  - Support multilabel: :ref:`Decision Trees <tree>`,
-    :ref:`Random Forests <forest>`, :ref:`Nearest Neighbors <neighbors>`.
-  - One-Vs-One: :class:`sklearn.svm.SVC`.
-  - One-Vs-All: all linear models except :class:`sklearn.svm.SVC`.
-
-Some estimators also support multioutput-multiclass classification
-tasks :ref:`Decision Trees <tree>`, :ref:`Random Forests <forest>`,
-:ref:`Nearest Neighbors <neighbors>`.
+  - Inherently multiclass:
+
+    - :class:`sklearn.ensemble.AdaBoostClassifier`
+    - :class:`sklearn.ensemble.BaggingClassifier`
+    - :class:`sklearn.naive_bayes.BernoulliNB`
+    - :class:`sklearn.calibration.CalibratedClassifierCV`
+    - :class:`sklearn.tree.DecisionTreeClassifier`
+    - :class:`sklearn.tree.ExtraTreeClassifier`
+    - :class:`sklearn.ensemble.ExtraTreesClassifier`
+    - :class:`sklearn.naive_bayes.GaussianNB`
+    - :class:`sklearn.gaussian_process.GaussianProcessClassifier`
+    - :class:`sklearn.ensemble.GradientBoostingClassifier`
+    - :class:`sklearn.neighbors.KNeighborsClassifier`
+    - :class:`sklearn.semi_supervised.LabelPropagation`
+    - :class:`sklearn.semi_supervised.LabelSpreading`
+    - :class:`sklearn.discriminant_analysis.LinearDiscriminantAnalysis`
+    - :class:`sklearn.svm.LinearSVC`
+    - :class:`sklearn.linear_model.LogisticRegression`
+    - :class:`sklearn.linear_model.LogisticRegressionCV`
+    - :class:`sklearn.neural_network.MLPClassifier`
+    - :class:`sklearn.neighbors.NearestCentroid`
+    - :class:`sklearn.svm.NuSVC`
+    - :class:`sklearn.linear_model.passive_aggressive.PassiveAggressiveClassifier`
+    - :class:`sklearn.linear_model.Perceptron`
+    - :class:`sklearn.discriminant_analysis.QuadraticDiscriminantAnalysis`
+    - :class:`sklearn.neighbors.RadiusNeighborsClassifier`
+    - :class:`sklearn.ensemble.RandomForestClassifier`
+    - :class:`sklearn.linear_model.RidgeClassifier`
+    - :class:`sklearn.linear_model.RidgeClassifierCV`
+    - :class:`sklearn.linear_model.SGDClassifier`
+    - :class:`sklearn.svm.SVC`
+
+  - Support multilabel:
+
+    - :class:`sklearn.tree.DecisionTreeClassifier`
+    - :class:`sklearn.tree.ExtraTreeClassifier`
+    - :class:`sklearn.ensemble.ExtraTreesClassifier`
+    - :class:`sklearn.neighbors.KNeighborsClassifier`
+    - :class:`sklearn.neural_network.MLPClassifier`
+    - :class:`sklearn.neighbors.RadiusNeighborsClassifier`
+    - :class:`sklearn.ensemble.RandomForestClassifier`
+    - :class:`sklearn.linear_model.RidgeClassifierCV`
+
+  - Support multiclass-multioutput:
+
+    - :class:`sklearn.tree.DecisionTreeClassifier`
+    - :class:`sklearn.tree.ExtraTreeClassifier`
+    - :class:`sklearn.ensemble.ExtraTreesClassifier`
+    - :class:`sklearn.neighbors.KNeighborsClassifier`
+    - :class:`sklearn.neighbors.RadiusNeighborsClassifier`
+    - :class:`sklearn.ensemble.RandomForestClassifier`
+
+  - One-Vs-One:
+
+    * :class:`sklearn.svm.SVC`.
+
+  - One-Vs-All:
+
+    * all linear models except :class:`sklearn.svm.SVC`.
```

Comment:
This is not true for multinomial logistic regression. So you need to make sure that you separate One-vs-all, One-vs-One and inherently multi-class. GradientBossting is also OvR.